src/api-request-handler.ts

```diff
@@ -68,7 +68,11 @@ export class ApiRequestHandler {
                 case 404:
                     throw new RevAiApiError(error);
                 case 403:
-                    throw new InsufficientCreditsError(error);
+                    if (error.response.data.error.includes("Not enough credits for payment")) {
```

Comment:
I'm not sure this ever actually gets triggered. It seems like now the submission still succeeds, the job will just fail. I don't know if it's always been this way...